error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/CharSequenceRulesTestInput.java

```diff
@@ -0,0 +1,16 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.common.collect.ImmutableSet;
+import tech.picnic.errorprone.refaster.test.RefasterRuleCollectionTestCase;
+
+final class CharSequenceRulesTest implements RefasterRuleCollectionTestCase {
+  ImmutableSet<Boolean> testCharSequenceIsEmpty() {
+    return ImmutableSet.of(
+        ((CharSequence) "foo").length() == 0,
```

Comment:
Maybe there is a better way, but now we know for sure its a `CharSequence` :D

Reply:
`new StringBuilder("foo")`, `str.subSequence(0, 1)`, ... :)